src/utils/timeformat.ts

```diff
@@ -0,0 +1,22 @@
+import dayjs from 'dayjs';
+
+export const formatTimeStr = (targetDay, platformDay, type: 'end' | 'start') => {
```

Comment:
This function is difficult to understand for me. Could you refactor the code for better understanding?